### DISCUSSION TITLE
Device support?

### DISCUSSION THREAD

**Discussion by mellofone** [jertel/vuegraf#104] [General]
Emporia makes a device to pair directly with a smart meter via Zigbee. It appears to use the same basic ideas as the 8 or 16 channel version. Is this supported and if not, would you like someone to be a tester?

**Comment by jertel** [comment]
I haven't attempted to use that device with Vuegraf. Have you checked with the pyemvue project, to see if they've proven the API can handle that device? If you're up for trying it, and providing detailed errors, log output, etc. then I think we have a good change of getting it up and running. But there could be a need for pyemvue changes, which will require those developers to assist.

**Comment by mellofone** [comment]
I did look through the PyEmVue project, but their documentation is a little thin on hardware specifics. The new device will arrive in a couple of days and I'll see what I can do with it out of the box.

**Comment by mellofone** [comment]
I got vuegraf running right out of the box with the Utility Connect model, polling data via the API and into influx with no issues whatsoever!

**Reply by jertel** [reply]
Great news! Thanks for reporting that. I've added a short [note](https://github.com/jertel/vuegraf#vue-utility-connect-energy-monitor) to the readme/docs mentioning that it's supported.

**Comment by mellofone** [comment]
Maybe I spoke too soon? Looking at the data in influx, it appears to be storing different values for two different items. In the field/tag **detailed**, there is a True/False option. Any ideas what the difference is? The values are quite different.

**Reply by jertel** [reply]
`detailed = True` represents backfilled per-second data that is queried from Emporia once every hour. 
`detailed = False` represents the per-minute average data that is collected every minute.

When building summative graphs in Grafana, you will normally want to filter on the detailed (either True or False) to avoid collecting duplicate data, since the per-minute data is an average of the more detailed per-second data covering the same period.

**Reply by mellofone** [reply]
Awesome, thanks. I now see that near the bottom of the README, I should have kept going!

**Reply by jertel** [reply]
Not your fault, I just added it to the README today. You were the second person this week to ask about that field.
